api/filrewardsd/client/client.go

```diff
@@ -0,0 +1,125 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	analyticspb "github.com/textileio/textile/v2/api/analyticsd/pb"
+	"github.com/textileio/textile/v2/api/filrewardsd/pb"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Client struct {
```

Comment:
This filrewards client will be used by hub to implement the all filrewards related APIs. That work is coming tomorrow.